app/javascript/components/editor/header/Settings.tsx

```diff
@@ -19,9 +19,9 @@ const WRAP = [
   { label: 'Off', value: 'off' },
 ]
 
-const TAB_ENABLED = [
-  { label: 'On', value: 'captured' },
-  { label: 'Off', value: 'default' },
+const TAB_MODE = [
+  { label: 'Editor', value: 'captured' },
+  { label: 'Accesibility', value: 'default' },
```

Comment:
```suggestion
  { label: 'Accessibility', value: 'default' },
```

I'd recommend **not** using `default` as the value, if that's possible. It's unclear from the code that by `default` it means "tab moves focus" and `captures` means "tab inserts whitespace". Both are names of *how* it's achieved and not *what* the mode represents.

Reply:
`default` is the codemirror key here. We could probably add another label of abstraction, but I think right now, that gets proxied straight to codemirror.

Reply:
I'd add it as a comment :P